legal-api/report-templates/continuationApplication.html

```diff
@@ -24,29 +24,30 @@
          </table>
          <div class="business-name-header">
             <label class="lbl-business-name">
-            {% if nameRequest.legalName %}
-               {{ nameRequest.legalName }}
-            {% else %}
-               {{ numberedDescription }}
-            {% endif %}
+               {% if business.legalName %}
+                  {{ business.legalName }}
+               {% elif nameRequest.legalName %}
+                  {{ nameRequest.legalName }}
+               {% else %}
+                  {{numberedDescription}}
+          {% endif %}
```

Comment:
why do we need this change?

Reply:
Current, it shows numbered company Description
![image](https://github.com/user-attachments/assets/cdef19d8-635f-4e2c-914f-d094c073088e)

Per new design, it shows the legal name if applicable:
![image](https://github.com/user-attachments/assets/29c5ae67-d12a-4cec-839f-2df3843acd3a)

After update:
![image](https://github.com/user-attachments/assets/ed93e902-8646-4a2f-9d7c-09e1c22348d8)

Reply:
Updated per new design:
when the numbered company didn't get a legal name yet, it shows numbered company Description;
After a numbered company gets a legal name (9900775 B.C. LTD), or it has a name request, it shows the legal name